### PR TITLE
OCD-3714: Authentication updates

### PR DESCRIPTION
**Pull request by andlar**

### CODE REVIEW

File: e2e/components/upload/upload-api-documentation/upload-api-documentation.spec.js [hunk 1]
```diff
@@ -12,16 +14,14 @@ beforeAll(async () => {
...
     loginComponent.logIn('admin');
   });
 
   it('can be uploaded successfully back to back', () => {
     uploadcomponent.uploadAPIDocFile('../../../resources/apiDoc/APIDoc_File.xlsx');
-    assert.include(uploadcomponent.apiDocUploadText.getText(),'was uploaded successfully.');
+    expect(uploadcomponent.apiDocUploadText.getText()).toContain('was uploaded successfully.');
     uploadcomponent.uploadAPIDocFile('../../../resources/apiDoc/APIDoc_File.xlsx');
-    assert.notInclude(uploadcomponent.apiDocUploadText.getText(),'was not uploaded successfully.');
+    expect(uploadcomponent.apiDocUploadText.getText()).toContain('was not uploaded successfully.');
```
Comment (palakpatel311) [inline e2e/components/upload/upload-api-documentation/upload-api-documentation.spec.js]: This assert was to **not** include so expect should be .not.toContain

File: e2e/components/upload/upload-listing/upload-listing.spec.js [hunk 2]
```diff
@@ -12,26 +14,28 @@ beforeEach(async () => {
...
     uploadListingComponent.uploadListing('../../../resources/upload-listing-beta/2015_WithCriteria.csv');
-    assert.include(uploadListingComponent.listingUploadText.getText(),'was not uploaded successfully');
+    expect(uploadListingComponent.listingUploadText.getText()).toContain('was not uploaded successfully');
   });
 
   it('can upload v19 template', () => {
     uploadListingComponent.uploadListing('../../../resources/listings/2015_v19_AQA1.csv');
-    assert.include(uploadListingComponent.listingUploadText.getText(),'was uploaded successfully. 1 pending products are ready for confirmation.', 'File has uploaded successfully');
+    expect(uploadListingComponent.listingUploadText.getText()).toContain('was uploaded successfully. 1 pending products are ready for confirmation.');
+    expect(uploadListingComponent.listingUploadText.getText()).toContain('File has uploaded successfully');
```
Comment (palakpatel311) [inline e2e/components/upload/upload-listing/upload-listing.spec.js]: This is additional check added which fails as line #32 is what we have on upload confirmation. #33 needs to be removed. same with #39

File: e2e/components/upload/upload-rwt/upload-rwt.spec.js [hunk 3]
```diff
@@ -12,23 +14,24 @@ beforeEach(async () => {
...
     loginComponent.logOut();
   });
 
   it('can upload valid format of rwt file', () => {
     uploadRwtComponent.uploadRwt('../../../resources/rwt/RWT_Upload_File.csv');
-    browser.waitUntil( () => !uploadRwtComponent.uploadButton.isDisplayed());
-    assert.include(uploadRwtComponent.fileUploadText.getText(),'was uploaded successfully. The file will be processed and an email will be sent to', 'File has uploaded successfully');
+    browser.waitUntil(() => !uploadRwtComponent.uploadButton.isDisplayed());
+    expect(uploadRwtComponent.fileUploadText.getText()).toContain('was uploaded successfully. The file will be processed and an email will be sent to');
+    expect(uploadRwtComponent.fileUploadText.getText()).toContain('File has uploaded successfully');
```
Comment (palakpatel311) [inline e2e/components/upload/upload-rwt/upload-rwt.spec.js]: Same comment as above.

File: e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js [hunk 4]
```diff
@@ -16,68 +17,67 @@ beforeAll(async () => {
...
-        for ( var k = 0; k < input.chplProductNumbers.length; k ++) {
+        let count = 0;
+        for (let k = 0; k < input.chplProductNumbers.length; k += 1) {
           if (fileContents.includes(input.chplProductNumbers[k])) {
-            count ++;
+            count += 1;
           }
         }
-        assert.equal(count,input.chplProductNumbers.length, 'All chpl product numbers aren\'t present in the download file');
+        expect(count, input.chplProductNumbers.length).toBe('All chpl product numbers aren\'t present in the download file');
```
Comment (palakpatel311) [inline e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js]: This assertion should be  expect(count).toBe(input.chplProductNumbers.length)
same applies to line #72,

File: e2e/pages/search/search.readonly.spec.js [hunk 5]
```diff
@@ -336,7 +336,7 @@ describe('the search page', () => {
       describe('the "Certification Date" filter', () => {
         it('should filter listing results', () => {
           page.moreFilterExpand(' View Certification Date ').click();
-          page.moreCertificationEndDateFilter.addValue('01/01/2019');
+          page.moreCertificationEndDateFilter.setValue(['01', 'Jan', 'Tab', '2019']);
```
Comment (palakpatel311) [inline e2e/pages/search/search.readonly.spec.js]: For some reason, dates tests are failing as this line is setting those as 01/dd/2019. May be it should be page.moreCertificationEndDateFilter.setValue(['01', '01', 'Tab', '2019']); ???

**Review by palakpatel311 — CHANGES_REQUESTED** [review]

File: e2e/pages/search/search.readonly.spec.js (hunk 5 above)
Comment (andlar) [reply e2e/pages/search/search.readonly.spec.js]: I ran this locally a few different times trying to get it to go wrong, and every time it continued to pass. I'd lean towards leaving it as is and checking again after our AQA has settled down a bit

File: e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js (hunk 4 above)
Comment (andlar) [reply e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js]: Line 72 doesn't have anything about a count that I could see

File: e2e/pages/search/search.readonly.spec.js (hunk 5 above)
Comment (palakpatel311) [reply e2e/pages/search/search.readonly.spec.js]: I'd like to keep this conversation unresolved. Hopefully we can check this in QA and before staging we can resolve it.

**Review by palakpatel311 — DISMISSED** [review]

File: e2e/pages/search/search.readonly.spec.js (hunk 5 above)
Comment (andlar) [reply e2e/pages/search/search.readonly.spec.js]: Yep. Leaving a conversation unresolved is a great way to make sure we handle it eventually. We're not supposed to merge any PRs to staging without having resolved each and every conversation, and I prefer to have the original author of the conversation be the one to resolve it, as I feel they're the person whose acceptance of the resolution we need to get

**Review by kekey1 — DISMISSED** [review]

**Review by palakpatel311 — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

**Review by tmy1313 — DISMISSED** [review]

**Review by palakpatel311 — COMMENTED** [review]
Impersonation tests are failing due to login change. Looks like missed those spec in this

File: package.json [hunk 6]
```diff
@@ -83,21 +82,21 @@
...
+    "@wdio/sync": "^7.11.1",
     "angular-mocks": "^1.7.2",
     "babel-eslint": "^8.2.6",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.2.2",
     "babel-plugin-angularjs-annotate": "^0.9.0",
     "browser-sync": "~2.9.11",
     "browser-sync-spa": "~1.0.3",
-    "chromedriver": "^91.0.0",
+    "chromedriver": "^92.0.0",
```
Comment (palakpatel311) [inline package.json]: chrome version is updated to 93. can you update driver to 93, too?

**Review by palakpatel311 — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**